bus-mapping/src/evm/opcodes.rs

```diff
@@ -530,10 +546,108 @@ fn dummy_gen_call_ops(
         (true, _) => Ok(vec![exec_step]),
         // 2. Call to account with empty code.
         (_, true) => {
-            state.handle_return()?;
+            state.handle_return(geth_step)?;
             Ok(vec![exec_step])
         }
         // 3. Call to account with non-empty code.
         (_, false) => Ok(vec![exec_step]),
     }
 }
+
+fn dummy_gen_create_ops(
+    state: &mut CircuitInputStateRef,
+    geth_steps: &[GethExecStep],
+) -> Result<Vec<ExecStep>, Error> {
+    let geth_step = &geth_steps[0];
+    let mut exec_step = state.new_step(geth_step)?;
+
+    let tx_id = state.tx_ctx.id();
+    let call = state.parse_call(geth_step)?;
+
+    // Increase caller's nonce
+    let nonce_prev = state.sdb.increase_nonce(&call.caller_address);
+    state.push_op_reversible(
```

Comment:
For all the `push_op_reversible` done here, the same comment applies as before: the write to the statedb is already done in `push_op_reversible` so before it we just need a read-only query.